test-cargo-miri/run-test.py

```diff
@@ -102,6 +102,11 @@ def test_cargo_miri_run():
         "run.subcrate.stdout.ref", "run.subcrate.stderr.ref",
         env={'MIRIFLAGS': "-Zmiri-disable-isolation"},
     )
+    test("`cargo miri r` (subcrate, no ioslation)",
+        cargo_miri("r") + ["-p", "subcrate"],
+        "run.subcrate.stdout.ref", "run.subcrate.stderr.ref",
+        env={'MIRIFLAGS': "-Zmiri-disable-isolation"},
+    )
```

Comment:
Instead of adding new tests, can you change one of the existing ones? Seems a bit wasteful to have an entire new test for this. (Same for `t`.)